content/docs/command-reference/exp/show.md

```diff
@@ -147,109 +119,147 @@ metric or param.
 > can find the actual source code.
 
 Let's say we have run 3 experiments in our project. The basic usage shows the
-workspace (Git working tree) and experiments derived from `HEAD`
-(`10-bigrams-experiment` branch in this case), and all of their metrics and
-params (scroll right to see all):
+workspace (Git working tree) and experiments derived from `HEAD` (`master`
+branch in this case), and all of their metrics and params (scroll right to see
```

Comment:
Great job updating the example @daavoo, thanks. It's basically a guide so I see why you're not sure what to do in the UG. I think you shouldn't worry about the guide except minimal updates to reflect this change. We have not finished moving some of this info from the ref to the guide anyway so we'll have to revisit this.